File: trunk/ivef-qt/IVEFschema.cpp

#include "IVEFschema.h"

Schema::Schema() {

}

Schema::Schema(const Schema &val) : QObject() {

}

Schema & Schema::operator=(const Schema &val) {

    return *this;
}

QString Schema::getElementFormDefault() const {

    return "qualified";
}

QString Schema::getTargetNamespace() const {

    return "urn:http://www.hitt.nl/XMLSchema/ISIS/1.1";
}

QString Schema::toXML() {

    QString xml = "<Schema";
    xml.append(">\n");
    xml.append( "</Schema>\n");
    return xml;
}

QString Schema::toString(QString lead) {

    QString str = lead + "Schema\n";
    return str;
}

File: tags/0_1_0_RC1/ivef-objc/Classes/IVEFSchema.m

#import "IVEFSchema.h"


@implementation IVEFSchema

- (id) init {
    self = [super init];
    if (self != nil) {
    }
    return self;
}

-(NSString *) getElementFormDefault {

    return @"qualified";
}

-(NSString *) getTargetNamespace {

    return @"urn:http://www.hitt.nl/XMLSchema/ISIS/1.1";
}

-(void) setAttributes:(NSDictionary *)attributeDict {

}

-(NSString *) XML {

    NSMutableString *xml = [NSString stringWithString:@"<Schema"];
    [xml appendString:@">\n"];
    [xml appendString: @"</Schema>\n"];
    return xml;
}

-(NSString *) stringValueWithLead: (NSString *) lead {

    NSMutableString *str = [[[NSMutableString alloc] init] autorelease];
    [str setString: [lead stringByAppendingString:@"Schema\n"]];
    return str;
}


@end


